__tests__/pages/signup.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { render, fireEvent, wait } from '@testing-library/react'
+import Signup from '../../pages/signup'
+
+describe('Signup Page', () => {
+  const submitSignup = jest.fn()
+  const props = { submitSignup }
+
+  test('Should render without crashing', () => {
+    const { getByTestId } = render(<Signup {...props} />)
+    getByTestId('email')
+    getByTestId('username')
+    getByTestId('password')
+    getByTestId('firstName')
+    getByTestId('lastName')
+  })
+
+  test('Should not submit values', async () => {
+    const { getByTestId } = render(<Signup {...props} />)
+    const submitButton = getByTestId('submit')
+    fireEvent.click(submitButton)
+    await wait(() => expect(submitSignup).not.toBeCalled())
+  })
+
+  test('Should submit values', async () => {
+    const { getByTestId } = render(<Signup {...props} />)
+    const emailField = getByTestId('email')
+    const usernameField = getByTestId('username')
+    const passwordField = getByTestId('password')
+    const firstNameField = getByTestId('firstName')
+    const lastNameField = getByTestId('lastName')
+    const submitButton = getByTestId('submit')
+
+    await wait(() =>
```

Comment:
Can you please consolidate all the `fireEvent.change` into a single `wait()`?

Reply:
@hwong0305 Done!